tests/jballerina-unit-test/src/test/resources/test-src/closures/closure.bal

```diff
@@ -666,20 +666,20 @@ function test30() returns int|string {
 function test31() returns int|string {
     int|string x = 5;
     var func1 = function () {
-        if (x is int && x > 3) {
-            x = x + 1;
-            if (x > 5) {
-               x = x + 10;
+        if (x is int && <int>x > 3) {
```

Comment:
IMO we can remove these test cases as these test cases are testing the type being narrowed, but we don't narrow this any more. So I think we should remove these and create negative test cases